rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -98,8 +101,62 @@ def __init__(
             self._trainer = trainer_class(**trainer_config)
             self._trainer.build()
 
+    def fit(
```

Comment:
Should we find a way to move this computation into the actor all together? One data transfer is probably better than many for each mini batch. 

Roughly an additional 10 ms for a small mini batch.

Reply:
I actually thought about it a little. But the problem will be loading a batch to the actor, and the learning on it. Does this remind of some horrible api? Load_on_batch, learn_on_batch? Only PPO has introduced this multi gradient updates in mini batch mode which makes things more complex. I have to think about this a bit more to come up with a compromise. I def. don't wanna create the same complex api of learn on batch and load batch.

Reply:
I can ask Chen later about the performance hit that we can incur. 

I also don't like that old api.

Reply:
Tagging @gjoliver to chime in.

Reply:
don't quite have the context. this API seems fine. what's wrong with having minibatch_size and num_iters here?

Reply:
The call to update inside that function is a call to an actor.

Reply:
oh I see. PPOTrainer should probably receive a big batch and do mini-batch splitting by itself?
we can keep TrainerRunner more generic with the same update() remote call for all these different Trainers.

Reply:
Yep

Reply:
Agreed. We should leave the HOW to the PPOTrainer and NOT do it on the Algorithm's side anymore (as we do right now via multi_gpu_train_one_step, which has the minibatch logic built-in and uses the load.../learn_on_loaded_batch... API.

Reply:
Let's merge this PR. Doing this right, requires another PR. I have added the TODO to this PR to make sure it's not lost.

Reply:
Agreed.